docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,313 @@
+# Plugins 
+
+Trino uses a plugin architecture to extend its capabilities and integrate with
+various data sources and other systems. Trino includes many plugins as part of
+the binary packages - specifically the `tar.gz` archive and the Docker
+container.
+
+Plugins implement some of the following capabilities:
+
+* [Connectors](/connector),
+* [Authentication types](security-authentication)
+* [Access control systems](security-access-control)
+* [Event listeners](admin-event-listeners)
+* Additional types, functions, and table functions
+* Block encodings
+* Resource group configuration managers
+* Session property configuration managers
+* Exchange managers
+
+All plugins are optional for your use of Trino because they support a specific
+functionality, that is potentially not needed for your use. Plugins are located
+in the `plugin` folder of your Trino installation and are loaded automatically
+during Trino startup.
+
+(plugins-downloading)=
+## Downloading
+
+Typically, downloading a plugin is not necessary because Trino binaries include
+many plugins as part of the binary packages.
+
+Every Trino release publishes each plugin as a ZIP archive to the [Maven Central
+Repository](https://central.sonatype.com/). Refer to [plugins-list] for details.
+The specific location is derived from the Maven coordinates of each plugin as
+defined in the `pom.xml` of the source code for the plugin.
+
+For example, the PostgreSQL connector plugin can be found in the
+`plugin/trino-postgresql` directory, and the `pom.xml` file contains the
+following identifier section:
+
+```xml
+<parent>
+    <groupId>io.trino</groupId>
+    <artifactId>trino-root</artifactId>
+    <version>470</version>
+    <relativePath>../../pom.xml</relativePath>
+</parent>
+
+<artifactId>trino-postgresql</artifactId>
+<packaging>trino-plugin</packaging>
+```
+
+The Maven coordinates are therefore `io.trino:trino-postgresql:470` with version
+or `io.trino:trino-postgresql` without version. Use this term for a [search to
+locate the
+artifact](https://central.sonatype.com/search?q=io.trino%3Atrino-postgresql).
+
+The coordinates translate into a path to the ZIP archive on the Maven Central
+Repository. Use this URL to download the plugin.
+
+```
+https://repo1.maven.org/maven2/io/trino/trino-postgresql/470/trino-postgresql-470.zip
+```
+
+Availability of plugins from other projects and organizations varies widely, and
+may require building a plugin from source.
+
+When downloading a plugin you must ensure to download a version of the plugin
+that is compatible with your Trino installation. Trino and the installed plugins
+must use the same version of Trino to be fully compatible. For example, a Trino
+plugin compiled for Trino 470 can not be used with older or newer versions of
```

Comment:
This is only true for plugins shipped with Trino itself, not any Trino plugin that can be downloaded

Reply:
Well.. it must be against the SPI of the use Trino version.. hence the sentence later .. and yes there is some wiggle room .. but I think we should ignore that .. you think the info later in this paragraph is not enough?

Reply:
Maybe I even need to point out the dependency and version ??

Reply:
There are no runtime checks for SPI compatibility by default, it's up to the plugin author to verify that. There are no other ways of checking compatibility than executing the code, that is running different queries.

Since every Trino release is a major version change, it's potentially breaking, so there are no guarantees, but a plugin can still work with versions different from which it was built for. Saying otherwise is misleading. Let's just say it's not recommended, not that it won't work.

Reply:
I simplified here as suggested and also added a Compatibility section in the dev guide that goes into more dirty details.